### DISCUSSION TITLE
Problema di valori vuoti nelle risposte API per "received_documents" (invoice_number, vat_number)

### DISCUSSION THREAD

**Discussion by morphey1** [fattureincloud/api#426] [General]
A partire dal 14 Marzo, stiamo riscontrando un problema nell'uso delle API di Fattureincloud, specificatamente quando si effettuano chiamate all'endpoint "received_documents". I valori di alcuni campi chiave nella risposta JSON, come "invoice_number" e "vat_number" nell'oggetto "entity", risultano essere vuoti, nonostante il file XML scaricabile presenti i dati corretti.

Questo comportamento inaspettato impatta significativamente il nostro processo di automazione e gestione documentale, poiché ci affidiamo a questi valori per elaborazioni successive. Evidenziamo che il problema si verifica solamente nelle chiamate API e non nei file XML corrispondenti, i quali contengono tutti i dati attesi.

Abbiamo verificato e confermato che le chiamate API sono state effettuate correttamente, seguendo la documentazione ufficiale di Fattureincloud, e che questo problema ha avuto inizio intorno al 14 Marzo. Non abbiamo apportato modifiche al nostro sistema che potrebbero aver influenzato la ricezione dei dati.

Sollecitiamo un'indagine su questo problema e una risoluzione tempestiva, in quanto sta influenzando negativamente i nostri processi aziendali. In attesa di un vostro riscontro, rimaniamo a disposizione per fornire ulteriori dettagli o dati di log che possano facilitare l'individuazione e la risoluzione del problema. Grazie per l'attenzione e l'assistenza.

**Comment by valmoz** [comment]
Buongiorno @morphey1 
i colleghi del supporto dovrebbero averle risposto.
le pregherei di darci conferma, in modo tale che il suo caso venga gestito con la giusta urgenza.

Grazie
Mauro

**Reply by valmoz** [reply]
Chiudo perché è gestita tramite supporto clienti
